workspace sir vivor/games/ttp.js

```diff
@@ -181,8 +181,9 @@ class TTP extends Games.Game {
 			}
 			strs.push(str + "</ul></div>");
 		}
-		player.say("Current hand: ");
-		Rooms.get('survivor').say("/pminfobox " + player.id + ", " + (start + strs.join("") + "</div></div>"));
+		if (!player.hasHand) return Rooms.get('survivor').say("/pmuhtml " + player.id + ", ttp, <strong><center>Current hand:</center></strong><br>" + (start + strs.join("") + "</div></div>"));
```

Comment:
Is this necessary (e.g., can we just always pmuhtmlchange, or is starting w/ pmuhtml first necessary?)

Reply:
Also, this shouldn't change how it appears in the pms, just makes it so that the old box will go away when they get a new hand, right?

Reply:
it is necessary i think because there will be no "ttp" to change?

Reply:
maybe it is not necessary after there is one uhtml of "ttp" is sent

Reply:
and yes to your second question

Reply:
didn't see it before